### DISCUSSION TITLE
Mitiq for mitigating noise in QML algorithms

### DISCUSSION THREAD

**Discussion by Agurk2** [unitaryfund/mitiq#2079] [Q&A] [answered]
Hi Mitiq,
My team and I have been working with QML for the past month or so and part of that is exploring whether Mitiq can be incorporated into PennyLane's QML package. Is it necessarily more complicated than just appending error mitigating code before/after the main QML algorithm? Thanks in advance!

**Comment by purva-thakre** [comment]
> Is it necessarily more complicated than just appending error mitigating code before/after the main QML algorithm? 

Could you please provide more details on the algorithm + what you are trying to do (in words, not code) ? 

Pennylane is available as [a frontend in mitiq](https://mitiq.readthedocs.io/en/stable/guide/frontends-backends.html#frontends).

**Comment by AbdullahKazi500** [comment]
@purva-thakre we want to add error mitigation especially ZNE In our QML Model which is CNN ,GANS or similar I have the basic coding done and I am facing issue with wrapping of the function for the code

**Reply by purva-thakre** [reply, marked as answer]
Have you looked at the examples/tutorials in the documentation?

https://mitiq.readthedocs.io/en/stable/examples/pennylane-ibmq-backends.html

https://mitiq.readthedocs.io/en/stable/examples/molecular_hydrogen_pennylane.html

https://mitiq.readthedocs.io/en/stable/guide/zne-1-intro.html#problem-setup

It might help if you provide a working sample of all that you have tried so far. Here's a helpful link: https://stackoverflow.com/help/minimal-reproducible-example
